CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## [1.8.0]
+
+- **Feature** - Add additional user field to results in `Pull Request` query
+- **Chore** - Update documentation
```

Comment:
just a suggestion to add a bit more detail. Example, New and updated documentation available on official website.